libs/dh/wholesale/feature-calculations/src/lib/create/create.component.html

```diff
@@ -21,162 +21,111 @@
   (closed)="onClose($event)"
 >
   @if (!showPeriodWarning) {
-    <ng-container *rxLet="gridAreasQuery.valueChanges; let result">
-      @if (result.data && result.data.gridAreas) {
-        <form
-          id="create-calculation"
-          class="grid"
-          novalidate
-          [formGroup]="formGroup"
-          (ngSubmit)="
-            latestPeriodEnd && formGroup.controls.calculationType.value === 'BALANCE_FIXING'
-              ? (showPeriodWarning = true)
-              : modal.close(true)
-          "
-        >
-          <!-- Quarterly Resolution Transition Error -->
-          @if (formGroup.controls.dateRange.errors?.resolutionTransition) {
-            <watt-validation-message
-              size="normal"
-              type="danger"
-              icon="danger"
-              message="{{
-                t('create.quarterlyResolutionTransitionError', {
-                  resolutionTransitionDate: resolutionTransitionDate | wattDate
-                })
-              }}"
-            />
-          } @else if (
-            latestPeriodEnd && formGroup.controls.calculationType.value === "BALANCE_FIXING"
-          ) {
-            <watt-validation-message
-              size="normal"
-              type="warning"
-              icon="warning"
-              message="{{
-                t('create.periodWarning', { latestPeriodEnd: latestPeriodEnd | wattDate })
-              }}"
-            />
-          }
-
-          <!-- Execution type -->
-          <div class="chips">
-            <label>{{ t("create.executionType") }}</label>
-            <watt-filter-chip
-              choice
-              name="executionType"
-              value="ACTUAL"
-              [selected]="true"
-              (selectionChange)="onExecutionTypeSelected($event)"
-            >
-              {{ t("executionTypes.ACTUAL") }}
-            </watt-filter-chip>
-            <watt-filter-chip
-              choice
-              name="executionType"
-              value="SIMULATION"
-              [disabled]="true"
-              (selectionChange)="onExecutionTypeSelected($event)"
-            >
-              {{ t("executionTypes.SIMULATION") }}
-            </watt-filter-chip>
-          </div>
-
-          <!-- scheduling -->
-          <div class="chips">
-            <label>{{ t("create.scheduling") }}</label>
-            <watt-filter-chip choice name="schedule" value="NOW" [selected]="true">
-              {{ t("scheduling.NOW") }}
-            </watt-filter-chip>
-            <watt-filter-chip choice name="schedule" value="SCHEDULED" [disabled]="true">
-              {{ t("scheduling.SCHEDULED") }}
-            </watt-filter-chip>
-          </div>
+    <form
+      id="create-calculation"
+      class="grid"
+      novalidate
+      [formGroup]="formGroup"
+      (ngSubmit)="
+        latestPeriodEnd && formGroup.controls.calculationType.value === 'BALANCE_FIXING'
```

Comment:
Suggestion: import and expose CalculationType enum as a public field, so you can use it in the template. To have type safety

Reply:
Good idea!
37008b8fc41d90453fbff87a6bf18607f67c9f07